go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/filecoin-project/go-fil-markets
 go 1.13
 
 require (
-	github.com/filecoin-project/dagstore v0.2.0
+	github.com/filecoin-project/dagstore v0.2.1-0.20210720172151-1ebca744487c
```

Comment:
Should this depend on `v2.0.0-beta1` tag?

Reply:
Have updated this.